app/views/gardens/show.html.haml

```diff
@@ -12,6 +12,12 @@
 .row
   .col-md-9
     %p.btn-group= render 'gardens/actions', garden: @garden
+    - if can?(:edit, @garden) && can?(:create, Photo)
```

Comment:
if this was in gardens/actions instead, would it appear in lots of logical places?

Reply:
Good call, I'll work on it!